docs/how-to-guides/advanced-user-guides/enable-api-audit-log-in-downstream-clusters.md

```diff
@@ -77,7 +103,88 @@ spec:
             rke.cattle.io/control-plane-role: 'true'
 ```
 
-For more information about cluster configuration, refer to the REK2 or K3s cluster configuration reference pages.
+:::tip
+
+You can also use the directive `machineSelectorConfig` with proper machineLabelSelectors to achieve the same effect.
+
+:::
+
+For more information about cluster configuration, refer to the [RKE2 cluster configuration reference](../../reference-guides/cluster-configuration/rancher-server-configuration/rke2-cluster-configuration.md) pages.
+
+</TabItem>
+
+<TabItem value="K3s">
+
+:::note
+
+This feature is available in Rancher v2.7.2 and later.
+
+:::
+
+You can use `machineSelectorFiles` to deliver the audit policy file to the control plane nodes, and `machineGlobalConfig` to set the options on kube-apiserver.
+
+As a prerequisite, you must create a [secret](../new-user-guides/kubernetes-resources-setup/secrets.md) or [configmap](../new-user-guides/kubernetes-resources-setup/secrets.md) to be the source of the audit policy.
+
+The secret or configmap must meet the following requirements:
+
+1. It must be in the `fleet-default` namespace where the Cluster object exists.
+2. It must have the annotation `rke.cattle.io/object-authorized-for-clusters: <cluster-name1>,<cluster-name2>` which permits the target clusters to use it.
+
+:::tip
+
+Rancher Dashboard provides an easy-to-use form for creating the secret or configmap.
```

Comment:
Do we have a tutorial link or a "path" that would make easier to the the final user to use this tip?

Reply:
We do, I remember linking elsewhere to both pages yesterday. Let me add those

Reply:
fixed.